README-build-from-source.md

```diff
@@ -1,194 +0,0 @@
-This document describes how to build ATF, UEFI, Linux kernel and Android for HiKey960 from source.
```

Comment:
Why not keep the UEFI instructions with the UEFI source? Right now, nothing in the directory is connected to UEFI. Its all HiSi blobs.

Moving things to a build-from-source directory would be ok, but I think only once the blobs are being used.  Including documentation to build code that isn't being used is just confusing.